pkg/api/agentPoolOnlyApi/v20180331/validate.go

```diff
@@ -40,7 +40,7 @@ func (l *LinuxProfile) Validate() error {
 
 // Validate implements APIObject
 func (a *AADProfile) Validate(rbacEnabled *bool) error {
-	if rbacEnabled == nil || *rbacEnabled == false {
+	if rbacEnabled == nil || !*rbacEnabled {
```

Comment:
see `helpers.IsTrueBoolPointer()`, I think that's a more expressive way to put the above. E.g.:

`if !helper.IsTrueBoolPointer(rbacEnabled) {...`